Task_2_3_1/src/main/java/r/masud/ccfit/nsu/ru/Main.java

```diff
@@ -0,0 +1,127 @@
+package r.masud.ccfit.nsu.ru;
+
+
+import java.util.Scanner;
+
+public class Main {
```

Comment:
Even with only console version you should separate model, presenter and view layers